config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```
@@ -366,3 +366,23 @@ periodics:
     testgrid-alert-email: release-managers+alerts@kubernetes.io
     testgrid-dashboards: sig-release-releng-blocking
     testgrid-tab-name: git-repo-kubernetes-fast-forward
+- name: periodic-release-verify-image-signatures
+  interval: 4h
```

cluster field is required:

```suggestion
- name: periodic-release-verify-image-signatures
  cluster: k8s-infra-prow-build-trusted
  interval: 4h
```

---

Ah good catch thanks @ameukam  :)